src/Pages/ResList/StoreList.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import MenuItem from './MenuItem';
+import './StoreList.scss';
+
+const StoreList = () => {
+  const navigate = useNavigate();
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const menuListClick = e => {
+    searchParams.set('address', e.target.childNodes[0].data);
+    searchParams.set('menu', e.target.childNodes[2].data);
+    searchParams.set('category', '');
+    searchParams.set('limit', e.target.childNodes[4].data);
+    setSearchParams(searchParams);
+    navigate(`/resultlist?${searchParams.toString()}`);
+  };
```

Comment:
```suggestion
 const menuListClick = (address, menu, limit) => {
    searchParams.set('address', address);
    searchParams.set('menu', menu);
    searchParams.set('category', '');
    searchParams.set('limit', limit);
    setSearchParams(searchParams);
    navigate(`/resultlist?${searchParams.toString()}`);
  };
```
내부 요소를 일일히 찾아가는 방법 보다는 함수에 인자를 넘겨줘서 사용하도록 구현해주세요!